src/vs/workbench/parts/extensions/common/extensionsFileTemplate.ts

```diff
@@ -31,6 +41,12 @@ export const ExtensionsConfigurationInitialContent: string = [
 	'\t// See http://go.microsoft.com/fwlink/?LinkId=827846',
 	'\t// for the documentation about the extensions.json format',
 	'\t"recommendations": [',
+	'\t\t// List of extensions which should be recommended for users of this workspace.',
+	'\t\t// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp',
+	'\t\t',
+	'\t],',
+	'\t"unwantedRecommendations": [',
+	'\t\t// List of extensions which should not be recommended for users of this workspace. These extensions may be irrelevant, redundant, or otherwise unwanted.',
```

Comment:
A literal translation of this would mean that we are asking users to list ALL the extensions that they would consider irrelevant/redundant etc.

How about..

`List of extensions that will be skipped from the recommendations that VS Code makes for the users of this workspace...` ?